core/adapters/eth_tx.go

```diff
@@ -92,9 +91,10 @@ func createTxRunResult(
 		gasPrice.ToInt(),
 		gasLimit,
 	)
-	if IsClientRetriable(err) {
-		return pendingConfirmationsOrConnection(input)
-	} else if err != nil {
+	//if IsClientRetriable(err) {
+	//    return pendingConfirmationsOrConnection(input)
+	//} else
+	if err != nil {
```

Comment:
We ought to look here:

https://github.com/ethereum/go-ethereum/blob/master/core/tx_pool.go#L579

And decide what to do in case of errors.